### DISCUSSION TITLE
C# cefsahrp Chat window change detection

### DISCUSSION THREAD

**Discussion by hihihi1207** [cefsharp/CefSharp#4874] [General]
I'm in the middle of a program to make reading 'https://chzzk.naver.com/ ' easier. When the chat window says 'I'm a follower', I want to recognize the chat and send a notification to the messagebox

**Comment by amaitland** [comment]
https://github.com/cefsharp/CefSharp/wiki/Frequently-asked-questions#13-how-do-you-handle-a-javascript-event-in-c
